tests/wicked/aggregate/sut/t21_teaming_ifcfg_check_port_config.pm

```diff
@@ -0,0 +1,81 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: wicked teamd
+# Summary: Teaming, ifcfg - check port config
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
```

Comment:
generally I don't mind but I am not sure how it is accurate :) also I would at least mention another maintainer here :D @jlausuch

Reply:
sorry for bothering ;)